4.analyze_data/python_analysis_env.yml

```diff
@@ -17,4 +17,4 @@ dependencies:
   - conda-forge::umap-learn
   - conda-forge::matplotlib
   - pip:
-    - git+https://github.com/cytomining/pycytominer@afac3ea16818ad25f37318ecd5c5090c0eff5806
+    - pycytominer
```

Comment:
any reason to not use the pycytominer version on conda-forge?

Reply:
No particular reason, just what I have been implementing in other projects. Is there a reason this is not as good as conda-forge?

Reply:
It's generally best to avoid mixing pip and conda - it's ok for now (I don't want to inadvertently send you into a conda hell!) but probably best to implement moving forward.